lib/components/Snackbar/snackbar.scss

```diff
@@ -16,7 +16,7 @@
     background: var(--component-snackbar-bg);
     box-shadow: var(--core-elevation-shadow-330);
     position: fixed;
-    z-index: 999;
+    z-index: 149;
```

Comment:
Should it be controlled? Maybe for soma cases we need max z-index

Reply:
I don't really see anything that realistically uses any z-index between 149 and 999 so far, it's just the modal/portal that uses 150 and by this change I'm moving it behind it, so I'd say we don't need it now, and I'd add we shouldn't need this in the future if we properly set all z-indexes and not just 999, 9999, 9998, 99999, 100000, etc.

Reply:
Unless, there are already cases where we want to show snackbar above modal, which is a pretty ridiculous thing to do since we should have consistency between z-hierarchy of these components, starting from design implementation